Sources/FrontEnd/Parse/ParserState.swift

```diff
@@ -117,6 +117,11 @@ struct ParserState {
     lexer.sourceCode.range(startIndex ..< currentIndex)
   }
 
+  /// Returns an empty site starting and ending at `index`.
```

Comment:
```suggestion
  /// Returns a range starting and ending at `index`.
```

It's hard to explain, but the use of the word site here comes off a bit weird.  A site is usally _of_ or _for_ something.  This one is disembodied.  Also "empty site" is not really a thing, English-wise.